src/DotNetStatus/Views/Shared/_Layout.cshtml

```diff
@@ -54,8 +54,8 @@
             </div>
             <div class="navbar-collapse collapse">
                 <ul class="nav navbar-nav">
-                    <li><a asp-controller="Data" asp-action="Index">Data</a></li>
-                    <li><a asp-controller="Usage" asp-action="Index">Usage Data</a></li>
+                    <li><a asp-controller="Data" asp-action="Index">Tools Data</a></li>
+                    <li><a asp-controller="Usage" asp-action="Index">ApiPort Usage</a></li>
```

Comment:
Not sure if these titles are any clearer... maybe "Tools Coverage" and "ApiPort Results"?